TRB_label_assist/SVM_decision_boundaries.ipynb

```diff
@@ -264,9 +270,9 @@
     ],
     "metadata": {
         "kernelspec": {
-            "display_name": "Python 3",
+            "display_name": "emission-private-eval",
             "language": "python",
-            "name": "python3"
+            "name": "emission-private-eval"
```

Comment:
```suggestion
            "name": "python3"
```

Reply:
done.

Reply:
and so does this

Reply:
Reverted this back to Python 3 using git file history.